components/Wallet/EditWalletForm.tsx

```diff
@@ -27,7 +28,13 @@ export default function EditWalletForm ({ wallet, userPaybuttons, refreshWalletL
     if (params.name === '' || params.name === undefined) {
       params.name = wallet.name
     }
-    console.log('edit wallet params', params)
+    void await fetch(`/api/wallet/${wallet.id}`, {
```

Comment:
We should be using the `websiteDomain` environment variable for api requests.

Reply:
Addressed, also made sure this wouldn't be `undefined`.